tests/Support/Objects/UpdateExampleMutationWithInputType.php

```diff
@@ -49,7 +49,6 @@ public function args(): array
             ],
 
             'test_with_rules_nullable_input_object' => [
-                'name' => 'test',
```

Comment:
Why?

Reply:
don't know why they are there in the first place? does the name attribute do anything?

Reply:
> does the name attribute to anything

Well, yes 😄

You can use it to change the name from the key:
- key is a string, use an actual different name
- no explicit key (i.e. plain array), in which case the `name` is actually required

Back to this code: sure, might not be in use/make sense here but my point is: a PR should be focused and random clean ups should not be part of bigger feature like this.

We can totally remove all the `name` which aren't in use, probably a copypasta galore, but then it should be done a) consequently and b) separate from this PR.